compositor_api/src/input/whip.rs

```diff
@@ -18,6 +18,10 @@ pub struct WhipInput {
     /// Token used for authentication in WHIP protocol. If not provided, the random value
     /// will be generated and returned in the response.
     pub bearer_token: Option<Arc<str>>,
+    /// Internal use only.
+    /// Overrides whip session id which is used when referencing the input via whip server.
+    /// If not provided, it defaults to input id.
+    pub override_whip_session_id: Option<InputId>,
```

Comment:
```suggestion
    pub override_whip_session_id: Option<Arc<str>>,
```

I'm not sure if session ID is not already an established term(it's private, so it does not matter much)

Consider `whip_session_id_override`  (noun instead of verb)